src/assets/text/helloworld.cwl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env cwl-runner
+
+# 1st-tool.cwl example from https://www.commonwl.org/user_guide/02-1st-example/
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: echo
+inputs:
+  message:
+    type: string
+    inputBinding:
+      position: 1
+outputs: []
+
+# 1st-workflow.cwl example from https://www.commonwl.org/user_guide/21-1st-workflow/index.html
+cwlVersion: v1.0
+class: Workflow
```

Comment:
Add a dummy example author metadata
https://www.commonwl.org/user_guide/17-metadata/index.html

Reply:
Might just be easier to just copy & use this example for CWL Tools (maybe just swap out the names & emails).